apps/web/components/Chat.tsx

```diff
@@ -60,30 +69,44 @@ export function Chat() {
     }
   }, [cookie, setCookie])
 
+  // TODO: This efefct can be an util hook
+  useEffect(() => {
+    const init = async () => {
+      try {
+        // we can also just use the poolWindowAI method here, and use window.ai directly down there as well
```

Comment:
what do you mean by pool?

Reply:
Argh I forgot to rename the function name in this comment. The client lib export the function `waitForWindowAI` which was originally named poolWindowAI